core/variant/binder_common.h

```diff
@@ -135,7 +153,13 @@ struct PtrToArg<char32_t> {
 template <typename T>
 struct VariantObjectClassChecker {
 	static _FORCE_INLINE_ bool check(const Variant &p_variant) {
-		return true;
+		using TStripped = std::remove_pointer_t<T>;
+		if constexpr (std::is_base_of<Object, TStripped>::value) {
+			Object *obj = p_variant;
+			return Object::cast_to<TStripped>(p_variant) || !obj;
```

Comment:
Isn't this supposed to be `!obj || Object::cast_to<TStripped>(p_variant)`? Getting EA_BAD_ACCESS errors at this line

Reply:
Yes, it is. `Object::cast_to<T>()` takes in an Object *, so both values will be the result of `p_variant->operator Object*()` and will be the same. So, if `obj` is a nullptr, we shouldn't attempt the cast.

Reply:
Related #84858
Would your change fix the same issue? Maybe it's better solution.

Reply:
No, it doesn't. This should be changed either way because it's semantically incorrect, but it doesn't fix the issues I was having in #86602